.github/workflows/release.yml

```diff
@@ -149,8 +149,8 @@ jobs:
             build/workspace/*/*.ledger/*
           if-no-files-found: ignore
 
-      - name: "Make .rpm (devel) Package"
-        id: make_rpm_devel
+      - name: "Make .rpm Package"
```

Comment:
I think for clarity, we should keep all the `devel/(devel)` naming, rather than switch to talking about "the rpm". It'll make the review shorter, and it's consistent with other Linux packages.